src/main/java/core/basesyntax/strategy/TransactionStrategy.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.StorageTransaction;
+
+public class TransactionStrategy {
+    public Transaction getTransaction(StorageTransaction transaction) {
```

Comment:
Use map with Operation as key and realization of `Transaction` an value instead of switch
Create this map in Main method and set it using constructor.